src/labels/CustomLabels.labels

```diff
@@ -4103,7 +4103,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>healthSolutionHHAccNoContacts</shortDescription>
-        <value>Consider deleting these unused Household object records to save space. If you have completed a conversion to the Household Account model and no longer need the data on the Household object records, you can safely delete them. View these Household object records by running the Empty Household Objects report in the NPSP Health Check reports folder.</value>
+        <value>Consider deleting these unused Household Account records to save space. View these Household Account records by running the &lt;b&gt;Empty Household Accounts&lt;/b&gt; report in the NPSP Health Check reports folder.</value>
```

Comment:
@sam-knox , this change is verified fine (see screenshot below). There is another label with similar text, I am wondering if `healthSolutionHHObjNoContacts` requires any change?

Also, if this PR is referencing a WI, can you use `**lurch: attach W-....` to connect them?

<img width="1022" alt="Screen Shot 2019-07-17 at 8 47 24 PM" src="https://user-images.githubusercontent.com/28743286/61427683-f4caad80-a8d3-11e9-8b9c-56d1bda41b05.png">